### DISCUSSION TITLE
Add "Description" for each of the layers in an ArcRAVE Project

### DISCUSSION THREAD

**Discussion by AldenShallcross** [Riverscapes/riverscapes-website-jekyll#40] [Ideas]
It would benefit our staff if the description field for each layer in ArcRAVE (and Q/WebRAVE) is filled in or at least linked to a source with the description and associated details for the calculations.   For example, in the Unsuitable or Limited Opportunities Layer in BRAT projects: (a) qualitatively describe the layer and the information it is intended to convey to resource managers (with any caveats) and (b) what are the threshholds being used for each of the limiting factors.  When staff cite these data in their write-ups/analysis, they will often need to disclose the underlying assumptions.  Easy access to this information would help them do this.  It would also help them evaluate why the model estimated x or y.

![Add Description for Fields in ArcRAVE](https://user-images.githubusercontent.com/75643138/145472813-c52e760e-95f2-497c-92f7-06fbbf3966ed.JPG)

**Comment by MattReimer** [comment]
The good news here is that we have a mechanism for this but it's not where you were looking. In all the RAVE tools (Q,Arc,Web) you can right click on a layer and see metadata for it. In our most recent runs of RSContext you will see links to documentation about where that layer comes from etc.

 In cases where some general documentation is needed it will take the form of a link to a full-sized webpage that will give you more information about the layer you're looking at.

![Screen Shot 2021-12-10 at 7 33 27 AM](https://user-images.githubusercontent.com/1063391/145599969-fb6c4ab2-dd45-4fa3-8aea-71f94eabeeda.png)

The documentation process is ongoing and we don't have it for BRAT yet but I wanted to let you know that it's both possible and coming and where to find it when it does arrive.

**Reply by joewheaton** [reply]
@MattReimer I like the idea of referring users over to our own more verbose documentation we have. However, we should do a little homework on the industry standards that ESRI has used for years and make sure that a minimal set of stuff in our production grade tools is being exported and written to the standard places. We should be at least following industry standards to a minimum and then funneling off to our custom stuff where need be. See https://desktop.arcgis.com/en/arcmap/10.3/manage-data/metadata/what-is-metadata.htm

**Comment by jtgilbert** [comment]
Thanks for the continued feedback @AldenShallcross, we'll be using these ideas and suggestions as we make updates to both the model and the documentation!
